website/src/theme/MDXComponents/index.js

```diff
@@ -45,6 +45,7 @@ import Icon from '@site/src/components/icon';
 import Lifecycle from '@site/src/components/lifeCycle';
 import detailsToggle from '@site/src/components/detailsToggle';
 import expandable from '@site/src/components/expandable';
+import ConfettiTrigger from '@site/src/components/confetti/';
```

Comment:
If you'd like to use the component without having to import it in each docs file, you can add the `ConfettiTrigger` here to the `MDXComponents object below within this file.

Currently this import isn't added anywhere in the file causing the eslint warning